CHANGES/443.feature

```diff
@@ -0,0 +1,6 @@
+These changes fix https://github.com/aio-libs/aiohttp/issues/4714, that is,
```

Comment:
This is going to be a part of the changelog under the version.
It'd be weird to see something like:

v1.0.0
======
features
--------
* This changes fix xxx

Reply:
Because it's not a bug, there shouldn't be a mention of fixing things.
It may say something like "add support for XXX. resolves issue YYY"

Reply:
I used the fix terminology because it "fixes" an enhancement issue on `aiohttp`, but I've reworded it as "Allow" (I've searched the changelog and there are two other uses of this wording before).

Reply:
Right, things aren't very consistent here, unfortunately. But it shouldn't prevent us from trying to do better :)